app/src/main/java/org/onionshare/android/ShareManager.kt

```diff
@@ -142,6 +142,8 @@ class ShareManager @Inject constructor(
                 ShareUiState.ErrorStarting(true)
             }
 
+            TorState.Stopping -> error("Still observing TorState after calling stop().")
```

Comment:
If I understand this right, we shouldn't observe this state here because startSharingJob is cancelled and joined before we call TorManager#stop()?